package.json

```
@@ -0,0 +1,88 @@
+{
+  "name": "token-dashboard",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@chakra-ui/react": "^1.6.7",
+    "@emotion/react": "^11",
+    "@emotion/styled": "^11",
+    "@testing-library/jest-dom": "^5.11.4",
+    "@testing-library/react": "^11.1.0",
+    "@testing-library/user-event": "^12.1.10",
+    "@types/jest": "^27.0.1",
+    "@types/node": "^16.9.1",
+    "@types/react": "^17.0.20",
+    "@types/react-dom": "^17.0.9",
+    "framer-motion": "^4",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
+    "react-scripts": "4.0.3",
+    "typescript": "^4.4.3",
+    "web-vitals": "^1.0.1"
+  },
+  "scripts": {
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "test": "react-scripts test",
+    "eject": "react-scripts eject",
+    "lint": "eslint '*/**/*.{js,ts,tsx}' --quiet --fix",
```

I think we can separate this into two different scripts where one will only show the eslint issues and the other one will fix them:

```suggestion
    "lint": "eslint '*/**/*.{js,ts,tsx}'",
    "lint:fix": "eslint '*/**/*.{js,ts,tsx}' --quiet --fix",
```

We have similar code in the token dapp: 
https://github.com/keep-network/keep-core/blob/5d2a9f5361d924624994a68bc4f7efac9728feb6/solidity/dashboard/package.json#L48-L49

---

Similary to the code in dapp we can also add prettier scripts to execute with lint scripts:

```suggestion
    "lint": "eslint '*/**/*.{js,ts,tsx}'",
    "lint:fix": "eslint '*/**/*.{js,ts,tsx}' --quiet --fix",
    "format": "npm run lint && prettier --check .",
    "format:fix": "npm run lint:fix && prettier --write ."
```

Thoughts?

---

good idea. any issues using yarn here instead of npm?